src/script/deploy.js

```diff
@@ -145,22 +177,27 @@ async function getLineaStateBridgeAddress(config) {
   }
 }
 
+async function getWorldIDIdentityManagerAddress(config) {
+  if (!config.worldIDIdentityManagerAddress) {
+    config.worldIDIdentityManagerAddress = process.env.WORLD_ID_IDENTITY_MANAGER_ADDRESS;
+  }
+  if (!config.worldIDIdentityManagerAddress) {
+    config.worldIDIdentityManagerAddress = await ask(
+      `Enter WorldID Identity Manager Address: (${DEFAULT_WORLD_ID_MANAGER}) `,
+    );
+  }
+  if (!config.worldIDIdentityManagerAddress) {
+    config.worldIDIdentityManagerAddress = DEFAULT_WORLD_ID_MANAGER;
+  }
+}
+
 ///////////////////////////////////////////////////////////////////
 ///                            UTILS                            ///
 ///////////////////////////////////////////////////////////////////
-async function loadConfiguration(useConfig, environment) {
+async function loadConfiguration(useConfig) {
   if (!useConfig) {
     return {};
   }
-  const defaultConfigFile = `src/script/config/default-${environment}-config.json`;
-  if (!fs.existsSync(CONFIG_FILENAME)) {
-    if (fs.existsSync(defaultConfigFile)) {
-      fs.copyFileSync(defaultConfigFile, CONFIG_FILENAME);
-      console.log(`Default configuration for ${environment} copied to ${CONFIG_FILENAME}`);
-    } else {
-      console.warn(`Default configuration file for ${environment} not found.`);
-    }
-  }
```

Comment:
This code was made for storing default values inside small json files. I thought it's better than having it hardcoded into the deploy script or asking user each time.